app/Http/Controllers/Administration/SettingsController.php

```diff
@@ -22,6 +24,30 @@
 
 class SettingsController extends Controller
 {
+	/**
+	 * Set the Login information of the Lychee configuration
+	 * Either they are not already set and we directly bcrypt the parameters
+	 * or the current username and password are compared and changed if successful.
+	 *
+	 * To be noted this function will change the CONFIG table if used by admin
+	 * or the USER table if used by any other user
+	 *
+	 * @param SetAdminLoginRequest $request
+	 * @param SetLogin             $login
+	 *
+	 * @return void
+	 *
+	 * @throws LycheeException
+	 * @throws ModelNotFoundException
+	 */
+	public function setLogin(SetAdminLoginRequest $request, SetLogin $login): void
```

Comment:
IMHO the comment makes no sense. The request is `SetAdminLoginRequest`. Moreover, I doubt that methods changes the `configs` table.

Reply:
well spotted.